packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -144,6 +149,9 @@ const ApplicationNavigation = ({
   function updateDrawerIsOpen(value) {
     drawerMenuIsOpenRef.current = value;
     setDrawerMenuIsOpen(value);
+    if (onDrawerMenuStateChange) {
+      onDrawerMenuStateChange(value);
```

Comment:
Can we add a unit test for this?

Reply:
Added here 1376560be921551f9527d18db6352063b720f59d

Reply:
A jest test may have been more appropriate, but this works too.